### ISSUE TITLE
whitespace - missing whitespace

### ISSUE THREAD

**Issue by Zorgatone** [palantir/tslint#1013]
```TypeScript
let buttons: ionic.popup.IonicPopupButton[] = []; // Missing whitespace after equal sign
```

If I have two spaces after the equal sign the warning goes away. Weird, I should be able to put a single space

**Comment by JKillian** [comment]
Hi @Zorgatone, I can't duplicate this on my machine. What version of TSLint and what version of TypeScript are you using?
